### PR TITLE
HV: add whl-phx-i7 support

### PR DESCRIPTION
**Pull request by jsun26intel**
Add whl-phx-i7 configuration code and xml support

Tracked-On: #4998 
Signed-off-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: hypervisor/arch/x86/configs/whl-phx-i7/board.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
```
Comment (gvancuts) [inline hypervisor/arch/x86/configs/whl-phx-i7/board.c:2]:
```suggestion
 * Copyright (C) 2020 Intel Corporation. All rights reserved.
```

File: hypervisor/arch/x86/configs/whl-phx-i7/misc_cfg.h [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
```
Comment (gvancuts) [inline hypervisor/arch/x86/configs/whl-phx-i7/misc_cfg.h:2]:
```suggestion
 * Copyright (C) 2020 Intel Corporation. All rights reserved.
```

File: hypervisor/arch/x86/configs/whl-phx-i7/pci_devices.h [hunk 3]
```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
```
Comment (gvancuts) [inline hypervisor/arch/x86/configs/whl-phx-i7/pci_devices.h:2]:
```suggestion
 * Copyright (C) 2020 Intel Corporation. All rights reserved.
```

File: hypervisor/arch/x86/configs/whl-phx-i7/whl-phx-i7_acpi_info.h [hunk 4]
```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
```
Comment (gvancuts) [inline hypervisor/arch/x86/configs/whl-phx-i7/whl-phx-i7_acpi_info.h:2]:
```suggestion
 * Copyright (C) 2020 Intel Corporation. All rights reserved.
```

File: hypervisor/arch/x86/configs/whl-phx-i7/board.c (hunk 1 above)
Comment (gvancuts) [reply hypervisor/arch/x86/configs/whl-phx-i7/board.c:2]: Are these `*.c/*.h` fiels auto-generated? If so, we may need to update the tool that does it to include the correct year.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by jsun26intel** [comment]
the PR is updated with requested change.

**Comment by gvancuts** [comment]
> the PR is updated with requested change.

What about the link and the "Supported HW" table update?

**Pull request merged**